integration-tests/environments/electron/package.json

```
@@ -14,12 +14,13 @@
     "package": "electron-builder --dir",
     "runner:main": "node runner.js main",
     "runner:renderer": "node runner.js renderer",
-    "test": "npm run test:main && npm run test:renderer",
-    "test:main": "mocha-remote --id main -- concurrently npm:app-importer npm:runner:main",
+    "test": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter npm run test:main && npm run test:renderer",
```

I suggest moving this to the `test:renderer` script.

```suggestion
    "test": "npm run test:main && npm run test:renderer",
```